string robot_name       # "Robo1"
string result_msg       # 'All done'
string task_assignment  # '출고' or '입고'